<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_DialogFilterRule</class>
 <widget class="QDialog" name="C_DialogFilterRule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置过滤规则</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ModelingTool.qrc">
    <normaloff>:/ModelingTool/Images/conversionMap.png</normaloff>:/ModelingTool/Images/conversionMap.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="m_label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>    根据数据源属性字段取值进行过滤，当满足过滤条件时才能转换为对象。</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
       <item>
        <widget class="QListWidget" name="m_listWidget">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_groupBox">
         <property name="title">
          <string>操作符</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="m_btn_1">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>=</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="m_btn_2">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_btn_3">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="m_btn_4">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>&gt;=</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_btn_5">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;=</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="m_btn_6">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>&amp;&amp;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_btn_7">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>||</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="m_label_set">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Select * From </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="m_btn_clear">
           <property name="text">
            <string>清除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btn_save">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ModelingTool.qrc"/>
 </resources>
 <connections/>
</ui>
